src/platform/utilities/sso/keepAliveSSO.js

```diff
@@ -2,25 +2,53 @@ import * as Sentry from '@sentry/browser';
 
 import { ssoKeepAliveEndpoint } from 'platform/user/authentication/utilities';
 
-export default async function keepAlive() {
-  // Having outside of keepAlive method could expose information
+const logToSentry = data => {
+  let isCaptured = null;
+  const { message } = data;
   const caughtExceptions = {
     'Failed to fetch': {
-      logType: Sentry.Severity.Error,
+      LEVEL: Sentry.Severity.Warning,
     },
     'NetworkError when attempting to fetch resource.': {
-      logType: Sentry.Severity.Error,
+      LEVEL: Sentry.Severity.Warning,
     },
-    'The Internet connection appears to be offline': {
-      logType: Sentry.Severity.Info,
+    'The Internet connection appears to be offline.': {
+      LEVEL: Sentry.Severity.Info,
     },
     'The network connection was lost.': {
-      logType: Sentry.Severity.Info,
+      LEVEL: Sentry.Severity.Info,
     },
     cancelled: {
-      logType: Sentry.Severity.Info,
+      LEVEL: Sentry.Severity.Info,
+    },
+    default: {
+      LEVEL: Sentry.Severity.Error,
     },
   };
+
+  const LEVEL = caughtExceptions[message]
+    ? caughtExceptions[message].TYPE
+    : caughtExceptions.default.TYPE;
+
+  // Doesn't log "info" to Sentry
+  if (LEVEL !== Sentry.Severity.Info) {
```

Comment:
Would it make sense to either make this an env var or at least put at the top of the file, something like `SENTRY_LOG_THRESHOLD = Sentry.Severity.Info`, or `SENTRY_LEVELS_TO_LOG = [Sentry.Severity.Warn Sentry.Severity.Error]`, just wondering if we'd like a way to customize this in the future

Reply:
Yeah I think this is a good point for extensibility in the future.